shoop/themes/classic_gray/templates/classic_gray/shoop/front/macros.jinja

```diff
@@ -35,7 +35,7 @@
 {% endmacro %}
 
 {% macro render_cross_sell_products(product, relation_type="", title="") %}
-    {%- set cross_sell_products = shoop.product.get_product_cross_sells(product, relation_type) %}
+    {%- set cross_sell_products = shoop.product.get_related_products(product, relation_type) %}
```

Comment:
The default parameter for render_cross_sell_products macro does not seem correct anymore. Shouldn't it be "related"?